core/src/batch/mod.rs

```diff
@@ -247,25 +249,34 @@ mod tests {
     use crate::packets::ip::ProtocolNumbers;
     use crate::packets::Ethernet;
     use crate::testils::byte_arrays::{ICMPV4_PACKET, TCP_PACKET, UDP_PACKET};
+    use std::sync::mpsc;
 
     fn new_batch(data: &[&[u8]]) -> impl Batch<Item = Mbuf> {
```

Comment:
minor, but wonder if a test fn like this should be made public somewhere as a general helper? like when testing pipeline ideas in applications? maybe not, but thinking aloud.

Reply:
chances are we probably won't use this much since proptest random packet gen is preferred outside the crate. but yeah if we do end up needing this elsewhere, we can come up with a better name and move it into `testils` later.